components/bio-formats-plugins/src/loci/plugins/in/ImportProcess.java

```diff
@@ -494,13 +492,8 @@ private void initializeFile() throws FormatException, IOException {
     baseReader.setMetadataFiltered(true);
     baseReader.setGroupFiles(!options.isUngroupFiles() || options.isGroupFiles());
     if(options != null && !options.showROIs()){
-        MetadataOptions mo = baseReader.getMetadataOptions();
-        if(mo == null){
```

Comment:
Briefly reviewed the various usages within ImportProcess and the reader will usually be a concrete ImageReader subreader created by `createBaseReader`, the OmeroReader or a wrapper for virtualstack which should delegate to the concrete reader. As all of these instances should be extending `FormatReader` so we definitely expect a default options object to be initialized by default the usage.
Setting `metadataOptions` to `null` is certainly a scenario that I do not expect our stack to handle and we would likely have to review how to deal with it with concrete use cases.